docs/DeveloperGuide.md

```diff
@@ -272,13 +272,19 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 (For all use cases below, the **System** is `McGymmy` and the **Actor** is the `user`, unless specified otherwise)
 
-**Use case: List all food**
+
+**Use case: UC01 List food**
 
 **MSS**
+1. User requests to list food (UC01)
+2. McGymmy shows a list of food that user has added (UC02)
```

Comment:
I don't think this line need to add UC2?
Might be better to change it to:
2. McGymmy shows a list of food in the database.

Reply:
List of Food that user has added works too
LGTM